great_ape_safe/ape_api/badger.py

```diff
@@ -167,10 +167,7 @@ def claim_bribes_hidden_hands(self, claim_from_strat=True) -> dict:
         loop it thru them and returning the list of addresses
         and mantissas for latter processing.
         """
-        address = (
```

Comment:
why removing this? in case that in the future somehow another msig needs claiming from hh_data, no?

Reply:
Yeah, good point, will push an alternative: claim_for_strat option